Generate component-rollup-config dist via postinstall hook.

**Is your feature request related to a problem? Please describe.**
Problem is that we are not able to build packages that rely on `@ovh-ux/component-rollup-config`.

Assuming the following command:

```sh
$ lerna run --scope @ovh-ux/ng-ui-router-breadcrumb build
```

It reports the following error: 

```sh
$ lerna run --scope @ovh-ux/ng-ui-router-breadcrumb build
info cli using local version of lerna
lerna notice cli v3.18.1
lerna info versioning independent
lerna notice filter including "@ovh-ux/ng-ui-router-breadcrumb"
lerna info filter [ '@ovh-ux/ng-ui-router-breadcrumb' ]
lerna info Executing command in 1 package: "yarn run build"
lerna ERR! yarn run build exited 1 in '@ovh-ux/ng-ui-router-breadcrumb'
lerna ERR! yarn run build stdout:
yarn run v1.19.1
$ rollup -c --environment BUILD:production
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

lerna ERR! yarn run build stderr:
[!] Error: Cannot find module '@ovh-ux/component-rollup-config'
Error: Cannot find module '@ovh-ux/component-rollup-config'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/foo/bar/ovh/manager/packages/components/ng-ui-router-breadcrumb/rollup.config.js:7:36)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.require.extensions..js (/home/foo/bar/ovh/manager/node_modules/rollup/dist/bin/rollup:838:24)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)

error Command failed with exit code 1.

lerna ERR! yarn run build exited 1 in '@ovh-ux/ng-ui-router-breadcrumb'
```

**Describe the solution you'd like**
Use the `postinstall` hook to generate the `dist` folder after the first installation.

**Describe alternatives you've considered**
None.

**Additional context**
See the lerna `run` documentation: https://github.com/lerna/lerna/tree/master/commands/run#readme

--- a/packages/manager/tools/component-rollup-config/package.json
+++ b/packages/manager/tools/component-rollup-config/package.json
@@ -24,6 +24,7 @@
     "build": "tsc",
     "dev": "yarn run build",
     "dev:watch": "tsc -w",
+    "postinstall": "yarn run build",
     "lint": "run-p lint:ts lint:md",
     "lint:md": "eslint --quiet --fix --format=pretty --ext .md .",
     "lint:ts": "eslint --quiet --fix --format=pretty ./src/**/*.ts",

